src/libs/Navigation/AppNavigator/createCustomStackNavigator/index.js

```diff
@@ -25,17 +26,20 @@ const defaultProps = {
 };
 
 function ResponsiveStackNavigator(props) {
-    const isSmallScreenWidthRef = useRef(props.isSmallScreenWidth);
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    const isSmallScreenWidthRef = useRef(isSmallScreenWidth);
+
+    isSmallScreenWidthRef.current = isSmallScreenWidth;
+
```

Comment:
This solution was not effective and the same issue occurred again. https://github.com/Expensify/App/issues/31503